specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -312,6 +312,145 @@
         }
       }
     },
+    "/custom/models/{modelId}/copy": {
+      "post": {
+        "summary": "Copy Custom Model",
+        "description": "Copy custom model to user specified target Form Recognizer resource.",
+        "operationId": "CopyCustomModel",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "modelId",
+            "in": "path",
+            "description": "Model identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "copyRequest",
+            "in": "body",
+            "description": "Copy request parameters.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CopyRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Copy request is queued successfully.",
+            "headers": {
+              "Operation-Location": {
```

Comment:
Please add `"x-ms-long-running-operation": true,`
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation

Reply:
I'm wondering if we (should) need the extension when we have also documented the header. The presence of a Operation-Location implies that it is a long running operation, no?

Reply:
Added `"x-ms-long-running-operation": true,` to the `copy` API